src/ethereum/api/BaseGraphAPI.ts

```diff
@@ -62,6 +64,9 @@ export class BaseGraphAPI {
       const result = await this.client.query<T, TVariables>(options)
       return result
     } catch (error) {
+      if (isErrorWithMessage(error) && error.message.includes('aborted')) {
+        throw new HTTPError(error.message, {}, STATUS_CODES.error)
+      }
```

Comment:
Love the change! It's super important that we propagate the error when this happens.